starport-core/src/main/scala/com/krux/starport/util/notification/SNSNotification.scala

```diff
@@ -0,0 +1,15 @@
+package com.krux.starport.util.notification
+
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.db.record.Pipeline
+
+protected class SNSNotification extends Notification {
```

Comment:
this could be just an `object` instead of `class` too

Reply:
Agreed